addon-test-support/@ember/test-helpers/dom/-get-element.js

```diff
@@ -0,0 +1,22 @@
+import { getContext } from '../setup-context';
+
+export default function getElement(selectorOrElement) {
+  if (
+    selectorOrElement instanceof Window ||
+    selectorOrElement instanceof Document ||
+    selectorOrElement instanceof HTMLElement ||
```

Comment:
This can be simplified. At least, `HTMLElement` and `SVGElement` are both of subclasses of `Element`, so that is a simplification.
Also, both `Element` and `Document` are subclasses of `Node`, but I'm not sure we want to be so generic, as there is other things that inherit from it that might not be valid targets, like `DocumentFragment`s.

Reply:
Confirm. I plan to refactor to use `HTMLElement.prototype.nodeType` which should “just work”.